monkey/infection_monkey/post_breach/shell_startup_files/windows/modify_powershell_startup_file.ps1

```diff
@@ -0,0 +1,11 @@
+param (
+    [string]$startup_file_path = $profile
+)
+
+If (!(Test-Path $startup_file_path)) {  # create profile.ps1 file if it doesn't exist already
+    New-Item -Path $startup_file_path -ItemType "file" -Force
+}
+Add-Content $startup_file_path "# Successfully modified $startup_file_path" ;  # add line to $Profile
+cat $startup_file_path | Select -last 1 ;  # print last line of $Profile
+$OldProfile = cat $startup_file_path | Select -skiplast 1 ;
+Set-Content $startup_file_path -Value $OldProfile ;
```

Comment:
and if there was no `profile.ps1` to begin with? how do you delete the dirs you created?